Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift

```diff
@@ -168,13 +170,12 @@ public final class SwiftLanguageServer: ToolchainLanguageServer {
   /// Updates the lexical tokens for the given `snapshot`.
   /// Must be called on `self.queue`.
   private func updateLexicalTokens(
```

Comment:
This should probably be called `updateSyntacticTokens` now because the new highlighting is slightly more complex than lexical.